NPB3.3.1/NPB3.3-OMP/include-gem5/gem5/asm/generic/m5op_flags.h

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Why is this now copied in when it wasn't before? Also, it would be better to use -I<gem5 dir>/include instead of copying in the header files.

Reply:
Discussed in this comment: https://github.com/darchr/npb-hooks/pull/4/files/c411006e5e7c4e21e0e2e9d030ab756827039fdf#r353544578